example/pkg/apis/miskatonic/v1beta1/scale_university_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
```

Comment:
nit: sync the boilerplate

Reply:
yeah. Since it is generated by boilerplate.go.txt, modify it will be more flexible.